core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -237,11 +232,9 @@ export class DotEditContentFormComponent implements OnInit {
      */
     private initializeForm() {
         this.form = this.#fb.group({});
-        this.$store.contentType().fields.forEach((field) => {
-            if (!isFilteredType(field)) {
-                const control = this.createFormControl(field);
-                this.form.addControl(field.variable, control);
-            }
+        this.$formFields().forEach((field) => {
```

Comment:
From the perspective of performance, I believe that adding controls for each field could cause performance issues. While it is not a serious problem, I suggest creating and grouping the controls beforehand. Once the group is created, it is possible to define it as follows:

 const formControls = this.$formFields().reduce((acc, field) => ({
    ...acc,
    [field.variable]: this.createFormControl(field)
}), {});

this.form = this.#fb.group(formControls);

Reply:
Done. Change to use reduce.